tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -145,6 +149,56 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+
+Container on Open Network And Port Forwarding Persist After Reboot
+    [Setup]     NONE
+
+    Set Test Environment Variables
+
+    Log To Console  Create Port Groups For Container network
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+
+    ${createcommand}=  catenate  SEPARATOR=\ \
+    ...  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME}
+    ...  --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT}
+    ...  --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    ...  --force=true --bridge-network=bridge --compute-resource=%{TEST_RESOURCE} --no-tlsverify --no-tls
+    ...  --container-network=open-net --container-network-firewall=open-net:open
+
+    ${output}=  Run  ${createcommand}
+
+    Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}  ${true}
+    Log To Console  Installer completed successfully: %{VCH-NAME}
+
+    # Create a container on the open network
+    ${open-running}=  Launch Container  vch-restart-open-running  open-net
+    ${open-exited}=  Launch Container  vch-restart-open-exited  open-net  ls
+
+    # Create nginx on the open network and check port forwarding
+    Launch Container With Port Forwarding  webserver  10000  10001  open-net
+
+    # Reboot VCH
+    Reboot VM  %{VCH-NAME}
+    Wiat For VCH Initialization  20x  10 seconds
+
+    # Check if the open container is persisted
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect ${open-running} | jq '.[0].State.Status'
```

Comment:
We should confirm that this container does _not_ present any port mappings (will have entries for _exposed_ ports but no `Binding` element). This is to cover the case where we ended up with 0-65535 ports listed when Open.